app/controllers/api/v1/nav_controller.rb

```diff
@@ -4,8 +4,7 @@ module Api
   module V1
     class NavController < ApplicationController
       def index
-        # addresses = JSON.parse(params.keys.first, symbolize_names: true)
-        render json: NavSerializer.new(NavFacade.nav_object(params[:start_address], params[:end_address]))
+        render json: NavSerializer.new(NavFacade.nav_object(params[:from], params[:to]))
```

Comment:
We just need to make sure this is how the request comes in instead of start_address and end_address!